sci-libs/oce/oce-0.18.3.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils eutils check-reqs multilib java-pkg-opt-2
+
+DESCRIPTION="Development platform for CAD/CAE, 3D surface/solid modeling and data exchange"
+HOMEPAGE="https://github.com/tpaviot/oce"
+SRC_URI="https://github.com/tpaviot/oce/archive/OCE-$PV.tar.gz"
+
+LICENSE="|| ( Open-CASCADE-LGPL-2.1-Exception-1.0 LGPL-2.1 )"
+SLOT="${PV}"
+KEYWORDS="~amd64 ~arm64 ~x86"
```

Comment:
Did you actually try building this package on `arm64` and `x86`?

Reply:
No I did not build on x86 nor arm64. I just carried over the arches from the older ebuild. However I know there are some members of the community that use them there.

Reply:
You shouldn't add keywords to an ebuilds unless you can confirm the build. The proper way to handle this situation is to mask the `oce` flag on `arm64` and `x86` and open a keywording bug on `sci-libs/oce`.

Reply:
I don't have hardware and I did not have the time to set up a VM for those arches. If I remove arm64 and x86 from this ebuild I have to remove it from all kicad packages too, since this is straight dependency.

Reply:
If I mask the `oce` flag 3D viewing will become impossible on those arches in KiCad AFAIK. Now it's another question how many users are there for those arches and this package, maybe it's not really an issue?

Reply:
The mask is temporary while the keywording is in progress. Users always have the option to unmask it. It's the proper way to proceed. https://devmanual.gentoo.org/ebuild-maintenance/maintenance-tasks/index.html